src/main/resources/db/changelog/data/add-user-admin-role.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
```

Comment:
Is there a reason to use so many migrations?
I would use 1 migration for adding all roles and 1 migration to add all users.

Reply:
done